module-2/backend/week_four.md

```diff
@@ -10,14 +10,49 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+
+stored data from a web site like what's in your shopping cart or links you have clicked on
+
 * What’s the difference between a session and a cookie?
+
+a session in your log in and specific to your user. a cookie is saved data outside of the session
+
 * What’s a flash and when do you want to use flashes?
+
+a flash is how you can display a temporary message like: 'successful login', 'logged out', etc.
+
 * Why do people say “HTTP is stateless”?
+
+because the cookies and sessions is what hold state on your computer
```

Comment:
not entirely...Because a stateless protocol does not require the server to retain session information or status about each communications partner for the duration of multiple requests. HTTP is a stateless protocol, which means that the connection between the browser and the server is lost once the transaction ends